examples/preprocessing/plot_rereference_eeg.py

```diff
@@ -44,18 +44,22 @@
 raw.set_eeg_reference([])
 evoked_no_ref = mne.Epochs(raw, **epochs_params).average()
 
-evoked_no_ref.plot(axes=ax1, titles=dict(eeg='EEG Original reference'))
+evoked_no_ref.plot(axes=ax1, titles=dict(eeg='EEG Original reference'),
+                   show=False)
 
 # Average reference. This is normally added by default, but can also be added
 # explicitly.
 raw.set_eeg_reference('average', projection=True)
 evoked_car = mne.Epochs(raw, **epochs_params).average()
 
-evoked_car.plot(axes=ax2, titles=dict(eeg='EEG Average reference'))
+evoked_car.plot(axes=ax2, titles=dict(eeg='EEG Average reference'), show=False)
 
 # Re-reference from an average reference to the mean of channels EEG 001 and
 # EEG 002.
 raw.set_eeg_reference(['EEG 001', 'EEG 002'])
 evoked_custom = mne.Epochs(raw, **epochs_params).average()
 
-evoked_custom.plot(axes=ax3, titles=dict(eeg='EEG Custom reference'))
+evoked_custom.plot(axes=ax3, titles=dict(eeg='EEG Custom reference'),
+                   show=False)
+
+fig.show()
```

Comment:
do we need `show=False` + `fig.show()` combo? Couldn't we just leave the default so that show is True and the figure gets displayed?

Reply:
Yes, because otherwise the plot stops after the first subplot.

Reply:
but your in the last subplot here, so you shouldn't need show=False, right?

Reply:
I actually like it better not to call `fig.show()` but to leave `show` arg undefined in evoked.plot (not use it in evoked.plot at all for the last axis) so that it evaluates to default of True. I think it is more consistent with the rest of mne examples/tutorials. Otherwise it may be little confusing for newcomers why all the plots use `show=False`. But its a minor thing, really.